R/prepare-admin-panel.R

```diff
@@ -60,7 +60,7 @@ get_time_per_day <- function(log_data) {
 
 get_actions_per_day <- function(log_data) {
   log_data %>%
-    dplyr::filter(!(.data$action %in% c("login", "logout"))) %>%
+    dplyr::filter(!(.data$action %in% c("login user", "logout user"))) %>%
```

Comment:
Updating actions with the new tags.

Reply:
Does it make sense to use `login user` and `logout user` ?

I'm regretting adding that suffix with a space. Consider finding and replacing that or we'll create a new issue just for that.